python/ml4ir/base/pipeline.py

```diff
@@ -399,17 +399,29 @@ def run(self):
                 if CalibrationKey.CALIBRATION in self.model_config:
                     if self.model_config[CalibrationKey.CALIBRATION]['key'] == \
                             CalibrationKey.TEMPERATURE_SCALING:
-                        temperature = self.model_config[CalibrationKey.CALIBRATION][
-                            CalibrationKey.TEMPERATURE] if \
-                            CalibrationKey.TEMPERATURE in\
-                            self.model_config[CalibrationKey.CALIBRATION] else 1.5
+                        kwargs = self.model_config[CalibrationKey.CALIBRATION][
+                            CalibrationKey.ARGS] if CalibrationKey.ARGS in self.model_config[
+                            CalibrationKey.CALIBRATION] else {}
+
                         results = relevance_model.calibrate(relevance_dataset=relevance_dataset,
                                                             logger=self.logger,
                                                             logs_dir_local=self.logs_dir_local,
-                                                            temperature_init=temperature)
+                                                            **kwargs)
```

Comment:
we went from something clear and explicit to `**kwargs` here. What is the benefit?

Reply:
`zip_output`? explicit is better than implicit. Let's change this back.

Reply:
@balikasg  I should add more args to the temperature scaling layer in future, specifically for its optimizer. if I do not use **kwargs at the top level ,`calibrate()` in `pipeline.py `and `temperature_scale()` in `relevance_model.py`, I have to pass tons of args to these function explicitly with redundant docstrings. However,  all the temperature scaling args have been explicitly defined in its right place  [here ](https://github.com/salesforce/ml4ir/blob/e8aabc6eb57a4fd3807933ebe1f1fb05d2a9e776/python/ml4ir/base/model/calibration/temperature_scaling.py#L222) and we have all of them there.